src/style/colors.ts

```diff
@@ -323,7 +337,31 @@ function colorStringValue(color: string | object) {
   return color?.toString();
 }
 
-function adjustSaturation(colors: string[], color: string) {
+function adjustAllSaturations(colors: string[], color: string, levels: number[]) {
```

Comment:
is `color: string` actually `selectedColor`\`currentColor`?

Reply:
Yes... chenged to 'baseColor' to be extra clear